piet-gpu-types/src/lib.rs

```diff
@@ -1,3 +1,12 @@
 pub mod encoder;
 pub mod ptcl;
 pub mod scene;
+pub mod test;
+
+pub struct f16(half::f16);
```

Comment:
So as written this has the problem that there's no way to construct a value. That could be fixed by changing it to `pub struct f16(pub half::f16);` but I'm also thinking of a different approach. That mostly involves adding this code to the module (see line 16 of piet-gpu-derive/derive.rs):

```rust
            trait HalfToLeBytes {
                fn to_le_bytes(&self) -> [u8; 2];
            }
            
            impl HalfToLeBytes for half::f16 {
                fn to_le_bytes(&self) -> [u8; 2] {
                    self.to_bits().to_le_bytes()
                }
            }
```